### PR TITLE
Customize learning rate of some estimators.

### PR DESCRIPTION
**Pull request by floffy-f**
All sklearn estimators in skwdro now have a LR option that can be set to a float, so that if adapt is set to None the AdamW LR is tuneable.

### CODE REVIEW

**Comment by floffy-f** [comment]
*Note: tests will pass on main on gh actions, I tried them.*

**Review by iutzeler — APPROVED** [review]
